dist/common/scripts/scylla_util.py

```diff
@@ -714,11 +794,15 @@ def non_root_disks(self):
         """Returns all attached disks but root. Include ephemeral and EBS devices"""
         return set(self._disks["ephemeral"] + self._disks["ebs"])
 
-    def ephemeral_disks(self):
```

Comment:
What about the callers? Don't they need adjustment?

Reply:
This patch has scylla_io_setup fix to follow api changes, and also posted PR to follow api changes on machine image:
https://github.com/scylladb/scylla-machine-image/pull/264

Reply:
It's not so good that we have cross-package API changes. We should consider moving this code to scylla-machine-image, I don't know if anything in scylla.git needs it.

Reply:
@avikivity Hmm, but (aws|gcp|azure)_instance class are referenced from both scylla-server package and scylla-machine-image package, and ephemeral_disks() is also called from scylla_io_setup, not just from scylla-machine-image.

Reply:
Maybe we can move IaaS part of scylla_io_setup to scylla-machine-image, though.
(Also other IaaS related code, too)

Reply:
But if we do that, manulally installed Scylla on IaaS does not included IaaS related code. Or need to install scylla-machine-image.